src/app/modules/zxing-scanner/browser-multi-format-reader.ts

```diff
@@ -0,0 +1,22 @@
+import { BrowserCodeReader } from './browser-code-reader';
+import { MultiFormatReader, DecodeHintType, BinaryBitmap, Result } from '@zxing/library';
+
+
+export class BrowserMultiFormatReader extends BrowserCodeReader {
+    private multiReader: MultiFormatReader;
```

Comment:
Let's rename this guy to reader, so it can override it's parent prop, I see no reason to keep both as they have the same value.

Reply:
I tried this but it's not possible:

```
Class 'BrowserMultiFormatReader' incorrectly extends base class 'BrowserCodeReader'.
  Types have separate declarations of a private property 'reader'.
```

Would be necessary to make `super.reader` `protected` instead of `private` and remove `multiReader` completely. Second problem is it's type. There is one usage where a cast to `MultiFormatReader` would be necessary.

Reply:
I don't see any problem in changing the access level from private to protected. This class is not on ZXing project, so we can design it the way we want. Also, I can't see where this cast would be written. 🤔